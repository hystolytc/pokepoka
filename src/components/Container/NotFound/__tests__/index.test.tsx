import React from 'react'
import '@testing-library/jest-dom/extend-expect'
import { render } from '@testing-library/react'
import { NotFound } from '../index'

const describeComponent = '<NotFound /> test case'

describe(describeComponent, () => {
  const { container } = render(<NotFound />)

  test(`Render ${describeComponent} correctly`, () => {
    expect(container).toMatchSnapshot()
  })
})