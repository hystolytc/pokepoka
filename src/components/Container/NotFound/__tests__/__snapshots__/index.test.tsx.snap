// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> test case Render <NotFound /> test case correctly 1`] = `
<div>
  <div
    class="css-zn5vyk"
  >
    <p
      class="css-147x4p9"
    >
      404
    </p>
    <p
      class="css-1b4xw04"
    >
      Woops, something went wrong!
    </p>
  </div>
</div>
`;
